### ISSUE TITLE
[v0.9.4.x] Plugin could not be activated because it triggered a fatal error.

### ISSUE THREAD

**Issue by vysegonzo** [szepeviktor/fix-w3tc#274]
> Fatal error: Cannot redeclare w3_instance() (previously declared in /home/public_html/wp-content/plugins/w3-total-cache/w3-total-cache-api.php:527) in /home/public_html/wp-content/plugins/w3-total-cache-fixed/inc/define.php on line 1512

Any solution to solve this?

**Comment by nigrosimone** [comment]
hi @vysegonzo you have installed both w3-total-cache and w3-total-cache-fixed. If you want use w3-total-cache-fixed you must delete w3-total-cache.

**Comment by amiga-500** [comment]
Greets @vysegonzo !

To expand on what @nigrosimone is saying ...

* **W3 Total Cache (Fixed)** is this repository's official plugin name -- lets call it _W3TC-fixed_.
* **W3 Total Cache** is the WordPress official plugin name -- lets call it _W3TC_.

When _W3TC-fixed_ is installed in your WordPress (WP) website it first checks to see if there is an existing _W3TC_ installed already.  If there is then a helpful warning message is displayed (see attached image below) alerting you that W3TC-fixed _**will not activate**_ until you deactivate W3TC.  When you deactivate W3TC you are then free to activate W3TC-fixed.  And everything is now in harmony.  You can even keep W3TC installed -- you don't have to delete it!

However, if you already have W3TC-fixed activated and then proceed to install W3TC and activate it as well then you will see the "_Fatal Error_" message that you got.

W3TC and W3TC-fixed share a lot of the same common files and naming convention internally.  The problem is W3TC is not aware to end gracefully (e..g block activation) when it sees conflicts, unlike W3TC-fixed which is very much aware.

## TL;DR Summary
You _can keep both W3TC and W3TC-fixed_ on your WP sites but only one can be activated at any given time.  When you attempt to activate W3TC-fixed while W3TC is already active you get a friendly alert (see image below) indicating the existing conflict and a graceful denial is seen.  But if you attempt to activate W3TC while W3TC-fixed is already active you get a chaotic Fatal error -- the message you got.

Please reply if you need more assistance.  We're always happy to help.

![z4](https://cloud.githubusercontent.com/assets/5191497/21756789/d61372e2-d5f3-11e6-9328-c1900f92aa4e.gif)
